pkgs/development/libraries/forge/default.nix

```
@@ -0,0 +1,54 @@
+{ stdenv, fetchFromGitHub, cmake, pkgconfig
+, arrayfire, expat, fontconfig, freeimage, freetype, boost
+, mesa, libGLU_combined, glfw3, SDL2, cudatoolkit
+}:
+
+let
+  version = "1.0.4";
+in stdenv.mkDerivation {
+  pname = "forge";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "arrayfire";
+    repo = "forge";
+    rev = "ref/tags/v${version}";
```

```suggestion
    rev = "v${version}";
```
I'm getting `unable to checkout` when testing locally